[Cache] Update default cache behavior (#11359)
cloudflare/cloudflare-docs: content/cache/concepts/cache-control.md

## Origin Cache-Control behavior

> In the following section, we list the directives and behavioral conditions associated with the enabling or disabling of Origin Cache-Control.

Reviewer: ```suggestion
The following section covers the directives and behavioral conditions associated with enabling or disabling Origin Cache-Control.
```
Trying to avoid "we".
